exercises/concept/interest-is-interesting/.docs/instructions.md

```diff
@@ -5,7 +5,7 @@ Each year, the balance of your savings account is updated based on its interest
 The interest rate your bank gives you depends on the amount of money in your account (its balance):
 
 - 3.213% for a negative balance (balance gets more negative).
-- 0.5% for a positive balance less than `1000` dollars.
+- 0.5% for a positive balance greater or equal to 0 and less than `1000` dollars.
```

Comment:
@andrerfcsantos You suggested the sentence above but wouldn't this be more accurate?
```suggestion
- 0.5% for a balance greater or equal to 0 and less than `1000` dollars.
```

Reply:
Well, I guess this is more accurante mathematically speaking and is also explicit about the number 0.

But now I realized that the other lines say:

```
- 1.621% for a positive balance greater or equal than `1000` dollars and less than `5000` dollars.
- 2.475% for a positive balance greater or equal than `5000` dollars.
```

Is the correct way **greater or equal to** or **greater or equal than**? English is not my first language, but I think it is the first one. Anyways, I think we should write all the sentences in the same way.

And maybe the correct way of saying this in english is **greater than or equal to**.

What do you guys think?

Reply:
@ee7 Thanks for the help! @lucassperez Would be great if you could apply the fixes as suggested by @ee7.

(@lucassperez As background info, I asked in our Slack channel for a native speaker to help us out and @ee7 stepped up, Andre and me are both not native speakers as well.)

Reply:
Applied the suggestions. Thanks for all the suggestions, everyone!